src/index.js

```diff
@@ -1,6 +1,16 @@
 const p = console.log;
 const puts = console.log;
 
+Number.prototype.times = function(callback) {
+  for (let i = 0; i < +this; i++) {
+    callback(i);
+  }
+  return;
```

Comment:
I think that if you're returning `undefined` you could leave it. A function always return `undefined` by default.

Reply:
Thanks, I'll change that